### DISCUSSION TITLE
User reviews rating issue

### DISCUSSION THREAD

**Discussion by Abhishek-eezylife** [cinemagoer/cinemagoer#365] [Show and tell]
@alberanid we found an issue in your repo on user review ratings. while pulling users reviews ratings it returns wrong rating because of wrong selector in the rules sections. we solved the issue wants to create a PR for this fix

**Comment by alberanid** [comment]
Hi @Abhishek-eezylife : sure, any PR is welcome, thanks!

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
